views/index.html

```diff
@@ -129,8 +129,26 @@
             <div id="notification" role="region" aria-label="Notification">
                 <a href="#"  aria-label="cancel" ><span></span><i class="icon-cancel-circled"></i></a>
             </div>
-            <div class="primary" >
-                <div class="bgStatus current">
+
```

Comment:
Something in the new CSS here seems to be crushing the BG section over to the left, at least in my Chrome/macOS.

![Screen Shot 2019-05-19 at 11 11 55 AM](https://user-images.githubusercontent.com/801582/57986219-f2102100-7a26-11e9-9f5d-394d5204e80d.png)

Reply:
Hmm, the only change to this file was supposed to be the links. Checking

Reply:
This should now be reverted, good catch